types/powerbi-visuals-tools/index.d.ts

```diff
@@ -0,0 +1,1372 @@
+// Type definitions for Powerbi-visuals-tools 1.11
+// Project: https://github.com/Microsoft/PowerBI-visuals-tools
+// Definitions by: Microsoft <https://github.com/Microsoft>
```

Comment:
I'd recommend including yourself on this line so you can be identified as an author in future PRs